frontend/src/app/component/phenotypic-detail/phenotypic-detail.module.ts

```diff
@@ -1,21 +1,21 @@
 import { CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
-import { RouterModule } from '@angular/router';
+import { PhenotypicDetailComponent } from './phenotypic-detail.component';
 import { AgeControlComponent } from './age-control/age-control.component';
 import { SearchFilterComponent } from './age-control/search-filter/search-filter.component';
-import { PhenotypicFeatureComponent } from './phenotypic-feature.component';
+import { SelectChipComponent } from './select-chip/select-chip.component';
```

Comment:
Hi @belkassaby This import (L#5) is not resolving on my machine. It's likely not been committed into the PR draft.

Reply:
Please see PR #16